applications/connectivity_bridge/src/modules/uart_handler.c

```diff
@@ -442,6 +446,16 @@ static bool app_event_handler(const struct app_event_header *aeh)
 		return false;
 	}
 
+	if (is_uart_baudrate_event(aeh)) {
+		const struct uart_baudrate_event *event =
+			cast_uart_baudrate_event(aeh);
+
+		set_uart_baudrate(event->dev_idx, event->baudrate);
+
+		/* return true to consume event */
```

Comment:
Why is the event consumed?

Reply:
It's not used by anything else. I guess it doesn't strictly need to be consumed.